overlord/devicestate/devicemgr.go

```diff
@@ -927,9 +928,19 @@ func (m *DeviceManager) ensureInstalled() error {
 
 	m.ensureInstalledRan = true
 
-	tasks := []*state.Task{}
+	var prev *state.Task
 	setupRunSystem := m.state.NewTask("setup-run-system", i18n.G("Setup system for run mode"))
-	tasks = append(tasks, setupRunSystem)
+
+	tasks := []*state.Task{setupRunSystem}
+	addTask := func(t *state.Task) {
+		t.Set("setup-run-system-task", setupRunSystem.ID())
```

Comment:
do we use this?

Reply:
no, this was a carry over from the previous approach, it can be removed